listenbrainz/webserver/static/js/src/utils/utils.tsx

```diff
@@ -466,7 +473,9 @@ const getAlbumArtFromListenMetadata = async (
     return images?.[0].src;
   }
   /** Could not load image from music service, fetching from CoverArtArchive if MBID is available */
-  const releaseMBID = getReleaseMBID(listen);
+  // directly access additional_info.release_mbid instead of using getReleaseMBID because we only want
+  // to query CAA for user submitted mbids.
+  const releaseMBID = listen.track_metadata.additional_info?.release_mbid;
```

Comment:
Perhaps for  clarity?
```suggestion
  const userSubmittedReleaseMBID = listen.track_metadata.additional_info?.release_mbid;
```